package.json

```diff
@@ -21,20 +21,31 @@
   },
   "bugs": "https://github.com/helmetjs/ienoopen/issues",
   "scripts": {
-    "pretest": "standard --fix",
-    "test": "mocha"
+    "pretest": "npm run lint",
+    "prepublishOnly": "npm run build",
+    "lint": "tslint --project . --fix",
+    "test": "jest --config test/jest-config.json",
+    "build": "tsc"
   },
+  "main": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
   "devDependencies": {
+    "@types/connect": "^3.4.32",
+    "@types/jest": "^24.0.9",
+    "@types/supertest": "^2.0.7",
     "connect": "^3.6.6",
-    "mocha": "^5.2.0",
-    "standard": "^12.0.1",
-    "supertest": "^3.3.0"
+    "jest": "^24.3.1",
+    "supertest": "^3.3.0",
+    "ts-jest": "^24.0.0",
```

Comment:
`ts-jest` seemed like the best way to do testing here, but let me know if there's a better way. I don't mind rewriting things from Mocha.

Reply:
ts-jest is what I use personally (eg Microsoft/types-publisher). Seems like a good choice. Typescript itself uses mocha with jasmine I think.